www.openfisca.fr/www.openfisca.fr.conf

```diff
@@ -0,0 +1,63 @@
+server {
+	listen 80;
+	server_name openfisca.fr;
+
+        location /.well-known {
+            root /tmp/renew-webroot;
+        }
+
+	location / {
+		return 301 https://www.$server_name$request_uri;
+	}
+}
+
+server {
+	listen 80;
+	server_name www.openfisca.fr;
+
+        location /.well-known {
+            root /tmp/renew-webroot;
+        }
+
+	location / {
+		return 301 https://$server_name$request_uri;
+	}
+}
+
+server {
+	listen 443 ssl;
+	server_name openfisca.fr;
+
+        ssl_certificate /etc/letsencrypt/live/openfisca.fr/fullchain.pem;
+        ssl_certificate_key /etc/letsencrypt/live/openfisca.fr/privkey.pem;
+
+	location / {
+		return 301 https://www.$server_name$request_uri;
+	}
+}
+
+server {
+	listen 443 ssl;
+	server_name www.openfisca.fr;
+
+        ssl_certificate /etc/letsencrypt/live/www.openfisca.fr-0001/fullchain.pem;
+        ssl_certificate_key /etc/letsencrypt/live/www.openfisca.fr-0001/privkey.pem;
+
+	access_log /var/log/nginx/www.openfisca.fr-access.log;
+	error_log /var/log/nginx/www.openfisca.fr-error.log;
+
+	location / {
+		proxy_pass http://localhost:2010;
+		proxy_set_header Host $http_host;
+		proxy_http_version 1.1;
+		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+	}
+
+	location /robots.txt {
```

Comment:
Should we change the robot config so it does not reference openfisca.fr but openfisca.org instead ?

Reply:
That can be discussed in #13, but we're not there yet ;).